### PR TITLE
Some changes to esx_sit

### PR DESCRIPTION
**Pull request by FedericoDeBona**
Changes:
- Animation of getting up
- Fixed if chair is occupied
- Fixes chair position before sitting down
- Disabled weapon wheel while sit
- If player can't sit because chair is unreachable, will force to sit down teleporting the player

### CODE REVIEW

**Comment by XvenDeR** [comment]
awesome!

File: client.lua [hunk 1]
```diff
@@ -78,37 +86,66 @@ Citizen.CreateThread(function()
...
+	FreezeEntityPosition(object, true)
+	PlaceObjectOnGroundProperly(object)
 	local pos = GetEntityCoords(object)
-	local objectCoords = pos.x .. pos.y .. pos.z
+	local objectCoords = string.sub(pos.x, 1, 9) .. string.sub(pos.y, 1, 9) .. string.sub(pos.z, 1, 9)
 
 	ESX.TriggerServerCallback('esx_sit:getPlace', function(occupied)
 		if occupied then
-			ESX.ShowNotification('Cette place est prise...')
+			ESX.ShowNotification('Questo posto è occupato')
```
Comment (FedericoDeBona) [inline client.lua:132]: This seat is taken

**Comment by trilliot** [comment]
The TP feature may cause players to access usually unreachable spaces and get stuck.
As depicted in these pictures: https://imgur.com/a/YYQ8FeF

**Comment by FedericoDeBona** [comment]
@trilliot you're right, I'm working on a fix for it

**Comment by FedericoDeBona** [comment]
@trilliot can you check now, I should have fixed it

**Comment by trilliot** [comment]
Nope, still the same problem with 0dd0cc9

**Comment by FedericoDeBona** [comment]
@trilliot please tell me that now works for you too

**Pull request merged**